pymongo/client_options.py

```diff
@@ -152,7 +152,7 @@ def _parse_pool_options(
     connect_timeout = options.get("connecttimeoutms", common.CONNECT_TIMEOUT)
     socket_timeout = options.get("sockettimeoutms")
     wait_queue_timeout = options.get("waitqueuetimeoutms", common.WAIT_QUEUE_TIMEOUT)
-    event_listeners = options.get("event_listeners")
+    event_listeners = cast(Sequence[_EventListener], options.get("event_listeners"))
```

Comment:
I believe this is `Optional[Sequence[_EventListener]]`.

Reply:
fixed, thanks for catching that!